src/test/java/mate/academy/dao/impl/RoleDaoImplTest.java

```diff
@@ -0,0 +1,59 @@
+package mate.academy.dao.impl;
+
+import java.util.Optional;
+import mate.academy.dao.RoleDao;
+import mate.academy.exception.DataProcessingException;
+import mate.academy.model.Role;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RoleDaoImplTest extends AbstractTest {
+    private RoleDao roleDao;
+    private Role adminRole;
+    private Role userRole;
+
+    @Override
+    protected Class<?>[] entities() {
+
+        return new Class[] {Role.class};
+    }
+
+    @BeforeEach
+    void setUp() {
+        roleDao = new RoleDaoImpl(getSessionFactory());
+        adminRole = new Role(Role.RoleName.ADMIN);
+        userRole = new Role(Role.RoleName.USER);
```

Comment:
you dont need to create a new Role before each method

Reply:
done